src/Components/DropDown.tsx

```diff
@@ -0,0 +1,83 @@
+import debounce from 'lodash.debounce';
+import { useCallback, useMemo, useState } from 'react';
+import { Person } from '../types/Person';
+
+type DropDownProps = {
+  people: Person[],
+  onSelected?: (person: Person) => void
+};
+
+export const DropDown: React.FC<DropDownProps> = ({
+  people,
+  onSelected = () => { },
+}) => {
+  const [active, setActive] = useState(false);
+  const [text, setText] = useState('');
+  const [value, setValue] = useState('');
+
+  const handlerActive = (condition: boolean) => {
+    setActive(condition);
+  };
```

Comment:
1. function names should start with a verb
2. use `prev` value
```suggestion
  const handleIsActive = () => {
    setActive(prev => !prev);
  };
```